test-suite/coq-makefile/coqdoc1/run.sh

```diff
@@ -65,4 +61,15 @@ sort -u > desired <<EOT
 ./test/html/test.test.html
 EOT
 (coqc -config | grep -q "NATIVE_COMPILER_DEFAULT=yes") || sed -i.bak '/\.coq-native/d' desired
-exec diff -u desired actual
+diff -u desired actual
+
+(cd "$(find tmp -name coq-test-suite)" && find .) | sort > actual
```

Comment:
Why do we need the 1st find?

Reply:
dunno, it was like that already no (see above). we want to jump to a directory we don't know where it is, and there do a find... any better idea is welcome.